src/ui/map.js

```diff
@@ -173,6 +173,7 @@ const defaultOptions = {
  *
  * @param {boolean} [options.hash=false] If `true`, the map's position (zoom, center latitude, center longitude, bearing, and pitch) will be synced with the hash fragment of the page's URL.
```

Comment:
the JSDoc can be updated here to show it also accepts a string type. See https://devhints.io/jsdoc